common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -95,6 +95,97 @@ object FaciaWidths {
   }
 }
 
+object GarnettFaciaWidths {
```

Comment:
Would switch from `FaciaWidths` to the updated `GarnettFaciaWidths ` have any knock on impact elsewhere?

Reply:
Yes: everywhere cards are used, i.e. fronts, tag pages and paid content fronts, but I'd say this is expected.